interview.md

```diff
@@ -11,41 +11,61 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+A DSL built to help poker players keep track of data at the poker table in real time. This dsl would help reinvent poker analyis for poker players. Right now player rely on their memory to write down what happened on any given hand durring game. They may wait till break to add a particular interesting hand into thier notes, or keep a mental note of how a player plays durring a hand, or even wait many hours until the event has stopped or they got busted out to write down things that happened at the table all off of memory. Instead image a DSL for poker players that allows them to keep track of poker related events at the poker table, and even provide them with statistics so they can make their best move.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This DSL is aproprite for poker player so that they can input events that happen at the table in real time and allows for more accurate data represention then when they add it to thier notes after the fact. Additionally I keeps all of a players notes, poker data, and suggestions all in one place. The need is for real time note / obervation taking while sitting at the poker table. It would allow the players to take accurate observation in real time but also not take away from their attention to the table. 
+
```

Comment:
I wonder if you'd have to design very carefully to make sure that inputting things into the DSL doesn't distract from the table. I would think that would also take attention.